components/rollup_node.md

```diff
@@ -0,0 +1,81 @@
+# Rollup Node
+
+The consensus module of Optimisc Ethereum.
+
+## Summary
+
+The Rollup Node is a consenus client that determines the latest state of the
+rollup. It reads state from the canon chain (L1) to compute the rollup state.
+
+## Components
+
+#### Feed Oracle
```

Comment:
I like `feed indexer` myself to describe this but strong opinion very weakly held.

Reply:
Yeah, I don't love `Feed Oracle`. I think `Feed Indexer` is a more appropriate name.